<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>674</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Cutie-pie Wallpaper Changer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/clock.png</normaloff>:/images/clock.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Source</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
      <item row="0" column="0">
       <widget class="QRadioButton" name="sourceImage">
        <property name="text">
         <string>Single Image</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="image">
          <property name="placeholderText">
           <string>Location of image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="imageBrowse">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="sourceImageList">
        <property name="text">
         <string>I&amp;mage List</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLineEdit" name="listfile">
          <property name="placeholderText">
           <string>Location of list file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="listfileBrowse">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="sourceFolder">
        <property name="text">
         <string>Fo&amp;lder</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLineEdit" name="fileFolder">
          <property name="placeholderText">
           <string>Location of folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fileFolderBrowse">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="sourceDrop">
        <property name="text">
         <string>Drop</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>Drop items here</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="sourceWeb">
        <property name="text">
         <string>Web</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
        <item>
         <widget class="QComboBox" name="webSource"/>
        </item>
        <item>
         <widget class="QStackedWidget" name="webSourcePages"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="slideshowGroup">
     <property name="title">
      <string>Slideshow</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Duration</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QSpinBox" name="hrs">
          <property name="maximum">
           <number>8760</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Hours</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="min">
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Minutes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sec">
          <property name="maximum">
           <number>60</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Seconds</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Background</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLineEdit" name="bgcolor">
          <property name="inputMask">
           <string>\#HHHHHH</string>
          </property>
          <property name="text">
           <string>#303030</string>
          </property>
          <property name="placeholderText">
           <string>Format should be #RRGGBB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bgcolorSelect">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Compose</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="multiply">
        <property name="text">
         <string>Multiply</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="scale">
        <item>
         <property name="text">
          <string>Scaled, Keep Proportions</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Scaled and Cropped</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tiled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Gravity</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="gravity">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>North</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NorthEast</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>East</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SouthEast</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>South</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SouthWest</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>West</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NorthWest</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Center</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Target&lt;br&gt;Resolution</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="targetWidth">
          <property name="minimum">
           <number>200</number>
          </property>
          <property name="maximum">
           <number>32768</number>
          </property>
          <property name="value">
           <number>1920</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="targetHeight">
          <property name="minimum">
           <number>200</number>
          </property>
          <property name="maximum">
           <number>32768</number>
          </property>
          <property name="value">
           <number>1080</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="targetScreen">
          <property name="text">
           <string>Screen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="targetDesktop">
          <property name="toolTip">
           <string>Only useful when KDE restores multi-desktop spanning</string>
          </property>
          <property name="text">
           <string>Desktop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="backendGroup">
     <property name="title">
      <string>Backend Options</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Runtime Folder</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Timeout</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="running">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Having this disabled is equivalent to &amp;quot;on first run only&amp;quot;.&lt;/p&gt;&lt;p&gt;With daily logins, equivalent to 1 wallpaper/day.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Enabled</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="folder">
        <property name="toolTip">
         <string>Point KDE slideshow here</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>Config folder</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/shm/wallpaper</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/tmp/wallpaper</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="xsetbg">
        <property name="text">
         <string>Call XSetBG too</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Legacy</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Plasma</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="plasmaDBus">
        <property name="text">
         <string>Update image-plugin wallpaper through DBus</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>&lt;b&gt;&lt;i&gt;Waifu Slideshow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>sourceImage</tabstop>
  <tabstop>image</tabstop>
  <tabstop>imageBrowse</tabstop>
  <tabstop>sourceImageList</tabstop>
  <tabstop>listfile</tabstop>
  <tabstop>listfileBrowse</tabstop>
  <tabstop>sourceFolder</tabstop>
  <tabstop>fileFolder</tabstop>
  <tabstop>fileFolderBrowse</tabstop>
  <tabstop>hrs</tabstop>
  <tabstop>min</tabstop>
  <tabstop>sec</tabstop>
  <tabstop>bgcolor</tabstop>
  <tabstop>bgcolorSelect</tabstop>
  <tabstop>multiply</tabstop>
  <tabstop>scale</tabstop>
  <tabstop>gravity</tabstop>
  <tabstop>targetWidth</tabstop>
  <tabstop>targetHeight</tabstop>
  <tabstop>targetScreen</tabstop>
  <tabstop>targetDesktop</tabstop>
  <tabstop>folder</tabstop>
  <tabstop>running</tabstop>
  <tabstop>xsetbg</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
